includes/admin/admin-bar.php

```diff
@@ -166,9 +180,74 @@ function edd_store_mode_admin_bar_print_link_styles() {
 		#wp-admin-bar-edd-store-menu .edd-mode-test {
 			color: #FF8C00;
 		}
+
+		#wpadminbar .edd-menu-notification-counter {
+			display: inline-block !important;
+			min-width: 18px !important;
+			height: 18px !important;
+			border-radius: 9px !important;
+			margin: 7px 0 0 2px !important;
+			vertical-align: top !important;
+			font-size: 11px !important;
+			line-height: 1.6 !important;
+			text-align: center !important;
+		}
+
+		#wpadminbar .edd-menu-notification-indicator {
+			float: right !important;
+			margin: 10px 0 0 !important;
+			width: 8px !important;
+			height: 8px !important;
+			border-radius: 4px !important;
+		}
+
+		#wpadminbar .edd-menu-notification-indicator:after {
+			display: block !important;
+			content: "";
+			position: absolute !important;
+			width: inherit !important;
+			height: inherit !important;
+			border-radius: inherit !important;
+			background-color: inherit !important;
+			animation: edd-menu-notification-indicator-pulse 1.5s infinite !important;
+		}
+
+		@keyframes edd-menu-notification-indicator-pulse {
+			0% {
+				transform: scale(1);
+				opacity: 1;
+			}
+			100% {
+				transform: scale(3);
+				opacity: 0;
+			}
+		}
+
+		#wpadminbar #wp-admin-bar-edd-upgrade a {
+			background-color: #00a32a;
+			color: #fff;
+			margin-top: 5px;
+		}
+
+		#wpadminbar #wp-admin-bar-edd-upgrade a:hover {
+			background-color: #008a20;
+		}
+
+		#wpadminbar .edd-menu-form-last {
+			border-bottom: 1px solid #3c4146 !important;
+			margin-bottom: 6px !important;
+			padding-bottom: 6px !important;
+		}
+
+		#wpadminbar .wp-ui-notification.edd-menu-notification-counter,
```

Comment:
Can these rules be added conditionally or prefixed with a front end body class? The front end admin bar seems to be consistently black no matter what my user profile is set to, but if, for example, my profile is set to sunrise, the notification is lost in the admin:

![image](https://user-images.githubusercontent.com/4853066/181324305-447241f7-ba97-4cba-b9fd-a38b6c0dbf40.png)

Reply:
```css
body:not(.wp-admin) #wpadminbar .wp-ui-notification.edd-menu-notification-counter,
body:not(.wp-admin) #wpadminbar .wp-ui-notification.edd-menu-notification-indicator
```
This might be sufficient for the selectors.

Reply:
Done in [3ff9e2b](https://github.com/awesomemotive/easy-digital-downloads/pull/9321/commits/3ff9e2b14c922c0a76618195c4d06f658d12cb2b)

Reply:
I'm just going to not load them at all if we're not in the admin.